### PR TITLE
task3-TransportationApp

### PR DESCRIPTION
**Pull request by MKotlobay**
Transportation App Task

### CODE REVIEW

File: c-sharp-apps-mark-kotlobay/TransportationApp/PublicVehicle.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace c_sharp_apps_mark_kotlobay.TransportationApp
+{
+    public class PublicVehicle
+    {
+        private int line = 0;
+        private int id = 0;
+        public int maxSpeed = 0;
```
Comment (yim222) [inline c-sharp-apps-mark-kotlobay/TransportationApp/PublicVehicle.cs]: Why did you change it to public? it should be protected.

**Comment by yim222** [comment]
Hey Mark, WD! 
Just you have made protected accessor into public (see the review).

Also you didn't do this task:


> At MonitorTransportation
> Public int CountPlains(PublicVehicle[] vehicles)  - מקבלת מערך של רכבים מכל הסוגים ומחזירה את המס' של המטוסי נוסעים שבהם. 
> 

Please fix and then merge. 

Keep up with the good work. I appreciate it.

**Comment by MKotlobay** [comment]
> Hey Mark, WD! Just you have made protected accessor into public (see the review).
> 
> Also you didn't do this task:
> 
> > At MonitorTransportation
> > Public int CountPlains(PublicVehicle[] vehicles)  - מקבלת מערך של רכבים מכל הסוגים ומחזירה את המס' של המטוסי נוסעים שבהם.
> 
> Please fix and then merge.
> 
> Keep up with the good work. I appreciate it.

Cant find it in you'r code (In slack) - copied it again and searched ever there. nothing
